data/gegno/gegno intro missions.txt

```diff
@@ -0,0 +1,331 @@
+			`	Much to your dismay, the shot almost immediately ricochets off of it, and you watch as the massive monster tumbles over you, just barely leaping into the air above. The organism plummets back into the ground behind you, and you fall to your back with the wind knocked out of you. Raising your sidearm again, you aim once more, this time with your life possibly on the line. The creature explodes out of the ground once again with its gaping mouth pointed upward to the sky. With shaking hands, you take aim once again, and your finger slowly starts to pull on the trigger.`
+			`	A very loud cannon-like "bang" follows, but it most certainly did not come from your weapon. The beast before you falls downward in a cloud of black smoke, blasting sand and rubble everywhere with a thud. Once you're able to come to your senses, you look up to see the alien marching out of the mouth of it and towards your direction, which has you stand up quickly in response.` 
+			choice
+				`	(Attempt to communicate with the alien.)`
+			`	At this point, you have already invested a great amount of your energy into the situation. With a last "thump" of their foot, the alien now stands before you, giving you a much better look at them. They are at least several feet taller than you, extremely bulky, and slightly hunched over. Heavy-looking metallics surround a majority of their body, and in any exposed areas are rough-looking patches of skin and hair. A great deal of facial hair covers their face, and above human-like eyes are a pair of horns curved inward towards you.`
+			`	You aren't sure how to begin a conversation, but start by trying to greet the alien in a formal manner. Awkward silence ensues, and the alien looks you up and down in an almost unenthused manner. They emit a very deep grumble, and with a loud snort, reach into a crevice in their armor to grab something. Pulling out what looks to be some sort of compass or map, they point to a location on it, gesture to themselves, then lower the device slowly to you. It seems they want to be taken to one of their species' planets.`
+			`	You hesitantly take the device and turn to make your way to your ship. There doesn't feel to be any emergency to the situation, but you still feel uncomfortable. Before heading out, you take one last glimpse of the enormous creature in the background. Its head is very reptilian, close to dragon-like, with numerous eyes and a mane of spikes. A few notably small appendages are found at either end of it, almost like a salamander, but its surface is definitely scaly. The various signatures on your radar earlier must have been others of their kind. Glaring at you, then at the creature, the alien utters a single intelligible word in a very deep voice:`
+			`	"Voordt."`
+				flee
```

Comment:
This part I still have yet to add. This is the last part of the intro mission I'm working on- we transport him, then talk to the Quarg, and that changes depending on whether or not you've met the Quarg already as well. I'll get to your other comments in a bit here- just skimming things at the moment because this review was a surprise to me :P